llms/README.md

```diff
@@ -111,6 +111,17 @@ Here are a few examples of Hugging Face models that work with this example:
 - [microsoft/phi-2](https://huggingface.co/microsoft/phi-2)
 - [mistralai/Mixtral-8x7B-Instruct-v0.1](https://huggingface.co/mistralai/Mixtral-8x7B-Instruct-v0.1)
 - [Qwen/Qwen-7B](https://huggingface.co/Qwen/Qwen-7B)
+- [pfnet/plamo-13b](https://huggingface.co/pfnet/plamo-13b)
+- [pfnet/plamo-13b-instruct](https://huggingface.co/pfnet/plamo-13b-instruct)
+    - Please note that the prompt must be formatted according to the format written in the Usage section of the follwoing page: [Usage](https://huggingface.co/pfnet/plamo-13b-instruct#usage)
```

Comment:
I'm wondering if this is not relevant anymore now that we default to use the Hugging Face tokenizer chat template? (See #338 )

Also it looks like the comment about Qwn needing the special EOS and trust remote code option was dropped in the rebase. Could you re-add that?

Reply:
@awni Thank you for pointing out it. I removed the unnecessary notes from README in: https://github.com/ml-explore/mlx-examples/pull/303/commits/0381d15da6df3b09fe26aeadb7023197b0cf5b10

Well, I think the description about the special EOS for Qwen model still remains here: https://github.com/ml-explore/mlx-examples/blob/0381d15da6df3b09fe26aeadb7023197b0cf5b10/llms/README.md?plain=1#L126-L138

Could you check this update again?

Reply:
My mistake, thanks for the update!